### ISSUE TITLE
Error when passing a null NetworkObject over an RPC

### ISSUE THREAD

**Issue by maxkratt** [FirstGearGames/FishNet#715]
**General**
Unity version: Unity 2022.3.22f1
Fish-Networking version: 4.3.7 Pro
Discord link: https://discord.com/channels/424284635074134018/1260153317674455083/1260160920219549768

**Description**
The following error is happening when you send null in an RPC instead of a valid NetworkObject or NetworkBehaviour:
```
PrefabId -8192 is out of range.
UnityEngine.Debug:LogError (object)
FishNet.Managing.Logging.LevelLoggingConfiguration:LogError (string) (at Assets/FishNet/Runtime/Managing/Logging/LevelLoggingConfiguration.cs:129)
FishNet.Managing.NetworkManager:InternalLogError (string) (at Assets/FishNet/Runtime/Managing/NetworkManager.Logging.cs:83)
FishNet.Managing.Object.SinglePrefabObjects:GetObject (bool,int) (at Assets/FishNet/Runtime/Managing/Object/PrefabObjects/SinglePrefabObjects.cs:38)
FishNet.Managing.NetworkManager:GetPrefab (int,bool) (at Assets/FishNet/Runtime/Managing/NetworkManager.QOL.cs:163)
PassNullNetworkObjectError.PassNull:RpcReader___Server_SendNullNob_3323014238 (FishNet.Serializing.PooledReader,FishNet.Transporting.Channel,FishNet.Connection.NetworkConnection)
FishNet.Managing.Server.ServerManager:ParseReceived (FishNet.Transporting.ServerReceivedDataArgs) (at Assets/FishNet/Runtime/Managing/Server/ServerManager.cs:787)
FishNet.Managing.Server.ServerManager:Transport_OnServerReceivedData (FishNet.Transporting.ServerReceivedDataArgs) (at Assets/FishNet/Runtime/Managing/Server/ServerManager.cs:661)
FishNet.Transporting.Tugboat.Tugboat:HandleServerReceivedDataArgs (FishNet.Transporting.ServerReceivedDataArgs) (at Assets/FishNet/Runtime/Transporting/Transports/Tugboat/Tugboat.cs:250)
FishNet.Transporting.Tugboat.Server.ServerSocket:IterateIncoming () (at Assets/FishNet/Runtime/Transporting/Transports/Tugboat/Core/ServerSocket.cs:518)
FishNet.Transporting.Tugboat.Tugboat:IterateIncoming (bool) (at Assets/FishNet/Runtime/Transporting/Transports/Tugboat/Tugboat.cs:209)
FishNet.Managing.Transporting.TransportManager:IterateIncoming (bool) (at Assets/FishNet/Runtime/Managing/Transporting/TransportManager.cs:705)
FishNet.Managing.Timing.TimeManager:TryIterateData (bool) (at Assets/FishNet/Runtime/Managing/Timing/TimeManager.cs:1017)
FishNet.Managing.Timing.TimeManager:IncreaseTick () (at Assets/FishNet/Runtime/Managing/Timing/TimeManager.cs:671)
FishNet.Managing.Timing.TimeManager:<TickUpdate>g__MethodLogic|98_0 () (at Assets/FishNet/Runtime/Managing/Timing/TimeManager.cs:342)
FishNet.Managing.Timing.TimeManager:TickUpdate () (at Assets/FishNet/Runtime/Managing/Timing/TimeManager.cs:332)
FishNet.Transporting.NetworkReaderLoop:Update () (at Assets/FishNet/Runtime/Transporting/NetworkReaderLoop.cs:28)

```

**Replication**
1. Import the example package
2. Open the Example scene
3. Press play and the error should appear

**Expected behavior**
No error. The error appears to be new in 4.3.7 or 4.3.6.

**Example File**
[PassNullNetworkObject.zip](https://github.com/user-attachments/files/16142331/PassNullNetworkObject.zip)

**Comment by emorris00** [comment]
Also getting this on 4.3.7 Pro

**Comment by FirstGearGames** [comment]
Resolved in 4.3.8
